src/reanimated2/core.ts

```diff
@@ -66,8 +66,9 @@ export function getSensorContainer(): SensorContainer {
 }
 
 export function registerEventHandler<T>(
-  eventHash: string,
-  eventHandler: (event: T) => void
+  eventHandler: (event: T) => void,
+  eventName: string,
+  emitterReactTag?: number
```

Comment:
```suggestion
  emitterReactTag = -1 
```
then we can remove `emitterReactTag.isUndefined() ? -1 : emitterReactTag.asNumber();` check from C++.